app/models/lesson.rb

```diff
@@ -2,4 +2,12 @@ class Lesson < ApplicationRecord
   has_one_attached :photo
   belongs_to :user
   has_many :bookings
+
+  validates :category, presence: true, inclusion in { in: %w(violin guitar singing piano yoga chess crossfit stretching football basketball handball golf swimming),
```

Comment:
better to create a constante (more readable and reusable in code for the form)

Reply:
done